pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -117,12 +117,7 @@ func init() {
 				// BuildController.BuildUpdater (OSClientBuildClient)
 				{
 					Verbs:     sets.NewString("update"),
-					Resources: sets.NewString("builds"),
```

Comment:
You can't do this here.  Doing this will break old controllers running against new masters, right?